flexmeasures/data/models/time_series.py

```diff
@@ -309,6 +313,9 @@ def chart(
         ] = None,
         most_recent_beliefs_only: bool = True,
         include_data: bool = False,
+        include_sensor_annotations: bool = False,
+        include_asset_annotations: bool = False,
+        include_account_annotations: bool = False,
         dataset_name: Optional[str] = None,
         **kwargs,
     ) -> dict:
```

Comment:
I always wanted a short notice what we mean by "chart" here. Maybe using ":returns:". It is a chart in the sense of a vega chart right?

Reply:
I clarified: it's a JSON string defining a vega-lite chart spec.